### PR TITLE
Add connector role

### PR DESCRIPTION
**Pull request by VadimZud**

### CODE REVIEW

File: playbooks/install_connector.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- hosts: "{{ playbook_hosts | default('ydb') }}"
```
Comment (blinkov) [inline playbooks/install_connector.yaml:1]: Let's always call it "fq_connector" / "fq-connector" and not just "connector", especially in user-visible places like playbook names, settings, etc.

File: roles/ydb_fq_connector/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    system: true
+
+- name: create fq-connector user
+  user:
+    name: ydb-fq-connector
+    group: ydb-fq-connector
+    system: true
+    comment: "YDB FQ connector Service Account"
+
+- name: install fq-connector and cli binaries
```
Comment (blinkov) [inline roles/ydb_fq_connector/tasks/main.yaml:48]: Let's add two more install options like with the main binaries: download official release by version number and build from source.

File: roles/ydb_fq_connector/templates/fq-connector.service [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Type=simple
+User=connector
+ExecStart={{ connector_dir }}/bin/{{ ydb_fq_connector_executable }} server -c {{ connector_dir }}/cfg/ydb-fq-connector.yaml \
+    --connector-port {{ ydb_fq_connector_config_obj.connector_server.endpoint.port + item.offset }}{% if ydb_fq_connector_config_obj.metrics_server is defined %} \
+    --metrics-port {{ ydb_fq_connector_config_obj.metrics_server.endpoint.port + item.offset }}{% endif %}{% if ydb_fq_connector_config_obj.pprof_server is defined %} \
+    --pprof-port {{ ydb_fq_connector_config_obj.pprof_server.endpoint.port + item.offset }}{% endif %}
+
+Restart=always
+RestartSec=1
+SyslogIdentifier=fq-conector
```
Comment (blinkov) [inline roles/ydb_fq_connector/templates/fq-connector.service:18]:
```suggestion
SyslogIdentifier=fq-connector
```

File: roles/ydb_fq_connector/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- name: flush handlers
+  meta: flush_handlers
+
+- name: start fq-connector instances
+  ansible.builtin.systemd:
+    name: "fq-connector-{{ ydb_dbname }}-{{ item.instance }}.service"
+    state: started
+    enabled: true
+  loop: "{{ ydb_dynnodes }}"
+  when: item.use_connector | default(false)
```
Comment (blinkov) [inline roles/ydb_fq_connector/tasks/main.yaml:88]:
```suggestion
  when: item.use_fq_connector | default(false)
```

File: playbooks/install_connector.yaml (hunk 1 above)
Comment (VadimZud) [reply playbooks/install_connector.yaml:1]: "ydb_fq_connector" / "ydb-fq-connector" will be used

File: roles/ydb_fq_connector/templates/fq-connector.service (hunk 3 above)
Comment (VadimZud) [reply roles/ydb_fq_connector/templates/fq-connector.service]: Changed to "SyslogIdentifier=ydb-fq-connector"

File: roles/ydb_fq_connector/tasks/main.yaml (hunk 4 above)
Comment (VadimZud) [reply roles/ydb_fq_connector/tasks/main.yaml]: Changed to "item.use_ydb_fq_connector"

File: roles/ydb_fq_connector/tasks/main.yaml (hunk 2 above)
Comment (artgromov) [reply roles/ydb_fq_connector/tasks/main.yaml]: Decided to add these options in separate PR.

**Pull request merged**